specs/create_password/01_main.rst

```diff
@@ -0,0 +1,159 @@
++-------+-----------------+-----------+------------+
+| issue | title           | author    | created    |
++-------+-----------------+-----------+------------+
+| 65_   | Create password | tschubotz | 2019-01-22 |
++-------+-----------------+-----------+------------+
+
+.. _65: https://github.com/gnosis/safe/issues/65
+
+Create password
+===============
+
+1. `Main`_
+2. `User Interface`_
+3. `External Communication`_
+4. Other_
+
+.. _Main:
+
+.. contents:: Table of Contents
+    :depth: 3
+
+1. Problem Definition
+---------------------
+
+- We need the user to set a password for the app so:
+    1. Nobody can open the app or confirm transactions without the password.
+    2. We can encrypt the private key of the device.
+- Most devices have a fingerprint sensor. Users should be able to use it
+  additionally to the password.
+- We need the user to agree to the Terms and Privacy Policy.
+
+2. Inputs
+-----------
+
+- New password
+    - Minimum 8 characters (>=8)
+    - At least 1 number
+    - At least 1 letter
+    - No more than 2 identical characters in a row
```

Comment:
"than 3"

Reply:
why 3? xx is allowed, xxx not. That's at least what the UI says inide the app